Gems/XR/Code/Tests/XRTest.cpp

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzTest/AzTest.h>
+#include "XRTest.h"
+
+#if !AZ_TRAIT_DISABLE_ALL_XR_TESTS
```

Comment:
In OpenXRVkTests.cpp the trait there begins with `O3DE`.  Here we should replace the `AZ` with `O3DE`.